edit/source-editor.js

```diff
@@ -100,6 +107,14 @@ function createSourceEditor(style) {
     }
   }
 
+  function getCurrentLinter() {
+    const name = prefs.get('editor.linter');
+    if (cm.getOption('mode') === 'stylus' && name === 'csslint') {
```

Comment:
I think we should also check for LESS

Reply:
Sounds plausible for the currently available preprocessors.
LESS is `text/x-less`.
Stylus is `stylus`.